### PR TITLE
Fixing small things

### PR DESCRIPTION
**Pull request by KatherineMichel**
## Description

* Commented out talk and og submission links on About page
* Updated OG page to indicate that notifications have been sent
* Added get invitation letter link to OG page (maybe link to Visa Support page would be preferable?)
* Consolidated text below ticket table into table (was there a reason to have the text below the table? almost all of the text was exact duplicate of table)
* A few typos/grammar fixes

### CODE REVIEW

**Review by tim-schilling — APPROVED** [review]
Looks good to me.

File: _pages/covid19.html [hunk 1]
```diff
@@ -24,7 +24,7 @@ <h2 class="demp">
     <div class="row">
         <div class="column medium-9 medium-centered">
             <p>
-              In order to keep conference attendees safe, DjangoCon US will have an indoor mask mandate and required proof of negative COVID test. We strongly recommend that people get fully vaccinated and boosted before the conference as well. If you are not familiar, we suggest you check out the <a href="https://www.cdc.gov/coronavirus/2019-ncov/travelers/travel-during-covid19.html">CDC's Domestic Travel guidelines to stay safe</a>.
+              In order to keep conference attendees safe, DjangoCon US will have an indoor mask mandate and require proof of negative COVID test. We strongly recommend that people get fully vaccinated and boosted before the conference as well. If you are not familiar, we suggest you check out the <a href="https://www.cdc.gov/coronavirus/2019-ncov/travelers/travel-during-covid19.html">CDC's Domestic Travel guidelines to stay safe</a>.
```
Comment (drewbrew) [inline _pages/covid19.html:27]:
```suggestion
              In order to keep conference attendees safe, DjangoCon US will have an indoor mask mandate and require proof of a negative COVID test. We strongly recommend that people get fully vaccinated and boosted before the conference as well. If you are not familiar, we suggest you check out the <a href="https://www.cdc.gov/coronavirus/2019-ncov/travelers/travel-during-covid19.html">CDC's Domestic Travel guidelines to stay safe</a>.
```

File: _pages/tickets.html [hunk 2]
```diff
@@ -83,14 +83,14 @@ <h2>Ticket Prices and Types</h2>
         <td>Online Only - Corporate</td>
         <td>$199</td>
         <td>$299</td>
-        <td><div>Companies with more than 5 employees</div><div>(including state agencies and nonprofits, if able)</div></td>
+        <td><div>Companies with more than 5 employees</div><div>(including state or government agencies and nonprofits, if able)</div></td>
```
Comment (drewbrew) [inline _pages/tickets.html:86]:
```suggestion
        <td><div>Companies with more than 5 employees</div><div>(including government agencies and nonprofits, if able)</div></td>
```

**Review by drewbrew — APPROVED** [review]
Looks good, just a couple of follow-on ideas

**Pull request merged**